test/test_serialization.py

```diff
@@ -82,6 +82,7 @@ def _test_serialization_data(self):
         t2 = torch.FloatTensor().set_(a[0].reshape(-1)[1:4].clone().storage(), 0, (3,), (1,))
         b += [(t1.storage(), t1.storage(), t2.storage())]  # 7
         b += [a[0].reshape(-1)[0:2].storage()]  # 8
+        b += [torch.rand(1).rename('test')]  # 9 -- test named tensors
```

Comment:
Does `self.assertEqual` check for name equality after this change? I'm not sure that it does; if it uses `torch.isclose` then I don't expect it to